src/modules/reservation/reservation.service.ts

```diff
@@ -4,47 +4,63 @@ import { omit } from 'radash'
 import { PrismaService } from '@/modules/prisma/prisma.service'
 
 import { ReservationCreateDTO } from './dto/ReservationCreate.dto'
+import { ReservationResponseDTO } from './dto/ReservationResponse.dto'
+import { ReservationResponseFullDTO } from './dto/ReservationResponseFull.dto'
 
 @Injectable()
 export class ReservationService {
     constructor(private prisma: PrismaService) {}
 
-    create(rawData: ReservationCreateDTO) {
+    create(rawData: ReservationCreateDTO): Promise<ReservationResponseFullDTO> {
         return this.prisma.$transaction(async (db) => {
             const ommited = omit(rawData, ['services'])
 
             const newReservation = await db.reservation.create({
                 data: ommited,
+                include: {
+                    services: { include: { service: true } },
+                    HousingUnit: true,
```

Comment:
Tem que ser alterado no schema do prisma
```suggestion
                    housingUnit: true,
```